packages/next/src/server/lib/patch-fetch.ts

```diff
@@ -64,6 +67,14 @@ export function validateTags(tags: any[], description: string) {
     } else {
       validTags.push(tag)
     }
+
+    if (validTags.length > NEXT_CACHE_TAG_MAX_ITEMS) {
+      console.warn(
+        `Warning: exceeded max tag count for ${description}, dropped tags:`,
+        tags.slice(i, tags.length).join(', ')
```

Comment:
```suggestion
        tags.slice(i).join(', ')
```

Nit pick: the end param is optional